spec/classes/rcs_spec.rb

```diff
@@ -0,0 +1,81 @@
+require_relative '../spec_helper'
+
+describe 'rcs', :type => :class do
+
+  it { is_expected.to compile.with_all_deps }
+  it { is_expected.to contain_class('rcs::config') }
+
+  context 'default parameters' do
+    it do
+      is_expected.to contain_file('/etc/default/rcS') \
+        .with_content(/TMPTIME=0/)
+        .with_content(/SULOGIN=NO/)
+        .with_content(/DELAYLOGIN=NO/)
+        .with_content(/UTC=YES/)
+        .with_content(/VERBOSE=NO/)
+        .with_content(/FSCKFIX=NO/)
```

Comment:
I thnk it's often easier to read failures of these are broken out to separate `it{}` blocks so that they're tested and reported independently.